types/googlemaps/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "npm-naming": false
+        "npm-naming": false,
+        "no-redundant-jsdoc-2": false
```

Comment:
This is already part of the other PR so what's this PR good for?

Reply:
I'll rebase the other if/when this is merged.

Reply:
You cannot add/remove rules to existing linting. Please fix these and if you really need to disable some error, disable it inline instead of disabling for whole package